src/frontends/onnx/frontend/src/core/graph.hpp

```diff
@@ -52,7 +53,7 @@ class Graph : public std::enable_shared_from_this<Graph> {
     virtual bool is_ov_node_in_cache(const std::string& name) const;
     virtual ov::Output<ov::Node> get_ov_node_from_cache(const std::string& name);
     OPENVINO_SUPPRESS_DEPRECATED_START
```

Comment:
should we remove `OPENVINO_SUPPRESS_DEPRECATED_START` and `OPENVINO_SUPPRESS_DEPRECATED_END` sections?

Reply:
next pr